tools/tertiary-analysis/decoupler/decoupler_pathway_inference.xml

```diff
@@ -0,0 +1,84 @@
+<tool id="decoupler_pathway_inference" name="Decoupler Pathway Inference" version="1.4.0+galaxy0" profile="20.05" license="MIT">
+    <description>
+        Pathway inference using the Decoupler.
+    </description>
+    <requirements>
+        <requirement type="package" version="1.4.0">decoupler</requirement>
+    </requirements>
+    <command>
+        python '$__tool_directory__/decoupler_pathway_inference.py'
+            -i '$input_anndata'
+            -n '$input_network_file'
+            --min_n "$min_n"
+            $use_raw
+            --output "inference"
+            $write_activities_path
+    </command>
+    <inputs>
+        <param name="input_anndata" type="data" format="h5ad" label="Input AnnData file" />
+        <param name="input_network_file" type="data" format="tabular" label="Input Network file" />
+        <param name="min_n" type="integer" min="0" value="5" label="Minimum of targets per source. If less, sources are removed" />
```

Comment:
Targets per source here sounds very jargony. Maybe simplify the label and add a little explanation in a `help= ` variable within this `<param >`.

Reply:
rewrote it